jxl/src/frame.rs

```diff
@@ -105,6 +105,23 @@ impl ReferenceFrame {
             saved_before_color_transform,
         })
     }
+    #[cfg(test)]
+    pub fn random(
+        width: usize,
+        height: usize,
+        num_channels: usize,
+        saved_before_color_transform: bool,
+    ) -> Result<Self> {
+        use rand::SeedableRng;
+        let mut rng = rand_xorshift::XorShiftRng::seed_from_u64(0);
```

Comment:
Let's pass the RNG in too.

Reply:
(in particular, as of now, this will make the 4 reference frames identical, which wouldn't catch some errors)